AstroCal/calendar.kv

```diff
@@ -0,0 +1,189 @@
+<DayMode>:
+	Label: 
```

Comment:
Having both files named Calendar can be confusing and cause issues when importing, maybe change the .kv file to be called Calendar_layout or something a little more descriptive

Reply:
.kv file must be name same as app class ,that is the naming convention as the program looks for a kv file named calendar.kv since the class is called CalendarApp.We have however renamed the calendar.py file to be called calendar_ui.py for there to be a distinction